cellengine/resources/fcs_file.py

```diff
@@ -142,10 +143,17 @@ def create(
         """
```

Comment:
Can you update the `fcs_files` docstring and the first sentence with the latest text from the API docs please? (https://docs.cellengine.com/api/#create_2)

Reply:
Re: second commit, that's a good point. I intended this as a catch-all so that we never send snake-cased params to CE. I thought this might be useful because it will prevent the need for `[to_camel_case(v) for v in body]`-style code everywhere we make a POST or a PATCH request. I can also add the `alter_keys()` line to `BaseAPIClient.prepare_params` (l. 15), converting all request parameters to camelCase as well. This would be useful, as i.e. `FcsFile.get_events` currently only  accepts camelCase kwargs, and that would allow snake case kwargs as well.


Lmk what you think; I can certainly scrap that commit and just to `to_camel_case` in `.create`.

Reply:
(Docstring comment is also pending.)

---

Sure, okay with me to keep the `alter_keys` call in `BaseAPIClient`.